snaps/build-and-release-aws-eks-snaps.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+#
+
+# This script will build and release snaps for AWS EKS.
+set -eux
+
+EKS_RELEASE="${EKS_RELEASE:-edge/eks.0}"
+KUBE_VERSION="${KUBE_VERSION:-$(curl -L https://dl.k8s.io/release/stable.txt)}"
+KUBE_ARCH="amd64"
+
+source utilities.sh
+MAIN_VERSION=$(get_major_minor $KUBE_VERSION)
+
+source utils/retry.sh
+
+rm -rf ./release
+git clone https://github.com/juju-solutions/release.git --branch rye/snaps --depth 1
+(cd release/snap
+  make KUBE_VERSION=$KUBE_VERSION KUBE_ARCH="$KUBE_ARCH" \
+    targets="kubectl kubelet kube-proxy"
+  make KUBE_VERSION=$KUBE_VERSION KUBE_ARCH="amd64" \
+    targets="kubernetes-test"
```

Comment:
If this is amd64 only and we want to prevent building it for other platforms, do we need to worry about not being able to run it? Should this be an error until we get kubernetes-test on other platforms or should we just not include it?